react/features/welcome/components/styles.js

```diff
@@ -42,8 +43,8 @@ export default createStyleSheet({
         alignSelf: 'center',
         borderRadius: 50,
         flex: 0,
-        height: 100,
-        width: 100
+        height: SIDEBAR_AVATAR_SIZE,
+        width: SIDEBAR_AVATAR_SIZE
```

Comment:
you can remove height, width and borderRadius from here, since the avatar code overrides them based on the size prop